editor/src/clj/editor/animation_set.clj

```diff
@@ -73,9 +73,12 @@
         parent-resources (map (fn [x] (:parent-resource x)) animation-info)
 
         animation-set-builder (Rig$AnimationSet/newBuilder)
-        animation-ids (ArrayList.)]
-    
-    (AnimationSetBuilder/buildAnimations paths bones streams parent-ids animation-set-builder animation-ids)
+        animation-ids (ArrayList.)
+        workspace (resource/workspace resource)
+        project-path (workspace/project-path workspace)
+        data-resolver (ModelUtil/createFileDataResolver project-path)]
```

Comment:
Temporary solution, just to see that the concept works with the editor.

Reply:
Perhaps annotate this as well if it's a temporary solution?